src/storage/common_storage.ts

```diff
@@ -44,8 +45,8 @@ export type Workspace = Module & {
 };
 
 export const MODULE_TYPE_WORKSPACE = 'workspace';
-export const MODULE_TYPE_OPMODE = 'opmode';
-export const MODULE_TYPE_MECHANISM = 'mechanism';
+export const MODULE_TYPE_OPMODE = 'OpMode';
```

Comment:
Please leave these as lowercase.

Reply:
Wny?   If we are using them to determine the class we are inheriting from, then we'll need another dictionary that maps the lower case module types to PascalCase.

Reply:
still not sure I understand why, but I went ahead and put them back as lowercase and did a conversion in the extended python generator

Reply:
I don't want these values changed because the moduleType is stored. See https://github.com/wpilibsuite/systemcore-blocks-interface/blob/b63e3a08e5b740461a0ca9dd799ec846a9fe4cbf/src/storage/client_side_storage.ts#L321